m/speciesAbundance/speciesAbundance.R

```diff
@@ -4,7 +4,7 @@
 defineModule(sim, list(
   name = "speciesAbundance",
   description = "Species abundance simulator",
-  keywords = c("species", "abundance", "gaussian", "spatial"),
+  keywords = c("species", "abundance", "gaussian", "spatial","dummyExample"),
```

Comment:
Can you plz change that to "dummy", "example") OR "dummy example")
To make the keywords are separated ;)